### PR TITLE
Add a doc on sharing cluster access credentials via kubeconfig

### LINKED ISSUE #5489: Documentation adding user
Hi, I can't find any documentation how I add more admin users so my team members can deploy pods.

### PR DESCRIPTION
**Pull request by j3ffml**
Closes  #5489

### CODE REVIEW

**Comment by satnam6502** [comment]
What if I only want to move the definition for a specific cluster?
What if on my desktop machine I create clusters A and B.
On my laptop I create clusters C and D.
Now I want A, B, C and D to be available to both my desktop and laptop.
How do I do that?

Thanks. And sorry for being such a pain.

File: docs/sharing-clusters.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+This config bundle lives in `$HOME/.kube/.kubeconfig`, and is generated
+by `cluster/kube-up.sh`. Sample steps for sharing `.kubeconfig` below.
+
+**1. Create a cluster**
+```bash
+cluster/kube-up.sh
+```
+**2. Copy .kubeconfig to new host**
+```bash
+scp $HOME/.kube/.kubeconfig user@remotehost:/path/to/kubeconfig
```
Comment (a-robinson) [inline docs/sharing-clusters.md]: It should probably be /path/to/.kubeconfig

**Comment by jlowdermilk** [comment]
> What if I only want to move the definition for a specific cluster?

Added a section to the doc on creating standalone `.kubeconfig` manually. You would need to create a kubeconfig file and copy that.

> What if on my desktop machine I create clusters A and B.
On my laptop I create clusters C and D.
Now I want A, B, C and D to be available to both my desktop and laptop.
How do I do that?

`kubectl` loads and merges config from the following locations (in order)
1. `--kubeconfig=path/to/kubeconfig` commandline flag
2. `KUBECONFIG=path/to/kubeconfig` env variable
3. `$PWD/.kubeconfig`
4. `$HOME/.kube/.kubeconfig`

If you create a cluster(s) on laptop, copy `laptop:$HOME/.kube/.kubeconfig` to
some unique location on your desktop, and
```
export KUBECONFIG=path/to/copied/.kubeconfig
```
Vice versa for clusters created on desktop. Details and examples for `.kubeconfig` loading/merging rules can be found in [kubeconfig-file.md](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubeconfig-file.md).

Hmm. Now I'm thinking I should add the above to the doc...

**Comment by jlowdermilk** [comment]
Added sections for manually generating `kubeconfig` and the merging example from @satnam6502's question. PTAL

**Comment by satnam6502** [comment]
Now I have one location in my filesystem for clusters I created "here" and other locations on my filesystem for clusters created "there". Brain auto-destruct.

**Comment by a-robinson** [comment]
LGTM. Is this alright with you as well, Satnam?

**Comment by satnam6502** [comment]
LGTM, thank you kindly.

**Pull request merged**